src/views/v2/modal/attendee-registration/notice/non-ar.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Modal: Attendee Registration > Notice > Non AR tickets.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/modal/attendee-registration/notice/non-ar.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1amp
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ * @var int $non_meta_count The number of tickets in cart, without meta fields.
+ */
+
+$notice_classes = [
+	'tribe-tickets__notice--non-ar',
+	'tribe-common-a11y-hidden' => empty( $non_meta_count ),
+];
+
+$this->template(
+	'components/notice',
+	[
+		'notice_classes' => $notice_classes,
+		'content'        => sprintf(
+			// Translators: %s: The HTML wrapped number of tickets.
+			esc_html_x(
+				'There are %s other tickets in your cart that do not require attendee information.',
+				'Note that there are more tickets in the cart, %s is the html-wrapped number.',
+				'event-tickets'
+			),
+			'<span id="tribe-tickets__non-ar-count">' . absint( $non_meta_count ) . '</span>'
+		),
+	]
+);
```

Comment:
Thinking of the type of call, which is a notice, I preferred to make it an independent template. We already have this type of thing in other places. And in case somebody wants to add a class to the notice (quite likely when customizing) they don't have to review the template changes whenever we make changes/update the parent template (most likely than updating the notice).